

In file included from kernel/sys.c:8:
include/linux/compat.h:98: error: `SIGEV_PAD_SIZE' undeclared here (not in a function)


---

 25-akpm/include/linux/compat.h |    1 +
 1 files changed, 1 insertion(+)

diff -puN include/linux/compat.h~compat_mq-fix include/linux/compat.h
--- 25/include/linux/compat.h~compat_mq-fix	2004-04-08 01:49:19.539180328 -0700
+++ 25-akpm/include/linux/compat.h	2004-04-08 01:49:19.541180024 -0700
@@ -13,6 +13,7 @@
 #include <linux/sem.h>
 
 #include <asm/compat.h>
+#include <asm/siginfo.h>
 
 #define compat_jiffies_to_clock_t(x)	\
 		(((unsigned long)(x) * COMPAT_USER_HZ) / HZ)

_
